### PR TITLE
Update Profile should not deal with rpms

### PR DESCRIPTION
**Pull request by utopiabound**
Also if configure_repo() provides an empty file_contents,
truncate the file and return.

Fixes Issue #140 

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

File: chroma_agent/action_plugins/agent_updates.py [hunk 1]
```diff
@@ -26,6 +26,11 @@ def configure_repo(filename, file_contents):
     full_filename = os.path.join(REPO_PATH, filename)
     temp_full_filename = full_filename + ".tmp"
 
+    if file_contents.strip() == "":
+        # Touch and truncate the file
+        open(full_filename, "w")
+        return agent_result_ok
+
     # this format needs to match create_repo() in manager agent-bootstrap-script
     file_contents = file_contents.format(
```
Comment (jgrund) [inline chroma_agent/action_plugins/agent_updates.py]: Do we need this to happen prior to checking the file_contents? I.E. is it safe to assume it's empty before this part?

File: chroma_agent/action_plugins/agent_updates.py [hunk 2]
```diff
@@ -26,6 +26,11 @@ def configure_repo(filename, file_contents):
     full_filename = os.path.join(REPO_PATH, filename)
     temp_full_filename = full_filename + ".tmp"
 
+    if file_contents.strip() == "":
```
Comment (jgrund) [inline chroma_agent/action_plugins/agent_updates.py]: Given we'll have an empty file, should we just not write a file here?

Comment (utopiabound) [reply chroma_agent/action_plugins/agent_updates.py]: We need the file to be present for `unconfigure_repo()` below.  This also removes the old contents if there was any.

File: chroma_agent/action_plugins/agent_updates.py (hunk 1 above)
Comment (utopiabound) [reply chroma_agent/action_plugins/agent_updates.py]: If the file `strip()`s to "" then doing a `.format()` wont' make it non-empty

File: chroma_agent/action_plugins/agent_updates.py (hunk 2 above)
Comment (jgrund) [reply chroma_agent/action_plugins/agent_updates.py]: But `unconfigure_repo` will return success in the case of the file not existing:

https://github.com/whamcloud/iml-agent/blob/ec2a08b529889cb373503e613a7e20bde91efa41/chroma_agent/action_plugins/agent_updates.py#L53-L54

In addition, we no longer create the repo if it's empty in phase 1 of server deploy, so it most likely will not be there anyway:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/72fcd9c2e3892cd49629b29123b7fb4b8d189439/agent-bootstrap-script.template#L188-L190

**Pull request merged**